src/types/AdInfo.ts

```diff
@@ -69,6 +75,11 @@ export type AdLoadFailedInfo = {
      */
     adUnitId: string;
 
+    /**
+     * The ID representing AdView.
```

Comment:
Same comment as above. I suggest using ChatGPT to generate grammatically correct API docs.

Reply:
We'll run ChatGPT to check the API docs in a separate PR because there are a lot of small changes.